modules/client-configuration/pages/supported-features-sle-micro.adoc

```diff
@@ -0,0 +1,165 @@
+[[supported-features-suse]]
+= Supported {micro} and {opensuse}{nbsp}{microos} Client Features
+
+[WARNING]
+====
+Support for {micro} and {opensuse}{nbsp}{microos} clients is provided as a technology preview for testing purposes, and not all features are fully functional at this stage.
+This feature is expected to be fully supported in a later version of {productname}.
+Do not use this feature on production systems.
+====
+
+[IMPORTANT]
+====
+The operating system you run on a client is supported by the organization that supplies the operating system.
+{micro} is supported by {suse}.
+{opensuse}{nbsp}{microos} is supported by the {suse} community.
+====
+
+
+The icons in this table indicate:
+
+* {check} the feature is available on both Salt and traditional clients
+* {cross} the feature is not available
+* {question} the feature is under consideration, and may or may not be made available at a later date
+* {trad} the feature is supported only on traditional clients
+* {salt} the feature is supported only on Salt clients.
+
+
+[cols="1,1", options="header"]
+.Supported Features on {micro} and {opensuse}{nbsp}{microos} Operating Systems
+|===
+
+| Feature
+| {micro} and {opensuse}{nbsp}{microos}
+
+| Client
+| {salt}
+
+| Operating system packages
+| {salt}
+
+| Registration
+| {salt}
+
+| Install packages
+| {salt}
+
+| Apply patches (requires CVE ID)
+| {salt}
+
+| Remote commands
+| {salt}
+
+| System package states
+| {salt}
+
+| System custom states
+| {salt}
+
+| Group custom states
+| {salt}
+
+| Organization custom states
+| {salt}
+
+| System set manager (SSM)
+| {salt}
+
+| Service pack migration
+| N/A
+
+| Basic Virtual Guest Management {star}
+| {cross}
+
+| Advanced Virtual Guest Management {star}
+| {question}
+
+| Virtual Guest Installation (Kickstart), as Host OS
+| {cross}
+
+| Virtual Guest Installation (image template), as Host OS
+| {question}
+
+| System deployment (PXE/Kickstart)
+| {question}
+
+| System redeployment (Kickstart)
+| {cross}
+
+| Contact methods
+| {salt}: ZeroMQ
+
+| Works with {productname} Proxy
+| {salt}
+
+| Action chains
+| {question}
+
+| Staging (pre-download of packages)
+| {question}
+
+| Duplicate package reporting
+| {salt}
+
+| CVE auditing (requires CVE ID)
+| {salt}
+
+| SCAP auditing
+| {question}
+
+| Package verification
+| {question}
+
+| Package locking
+| {salt}
+
+| System locking
+| {question}
+
+| Maintenance Windows
+| {question}
+
+| System snapshot
+| {cross}
+
+| Configuration file management
+| {salt}
+
+| Snapshots and profiles
+| {salt}: Profiles supported, Sync not supported
+
+| Power management
+| {question}
```

Comment:
I think Power Management depends more on the Hardware than on the OS.
Only a wrong ACL in the UI could prevent this working. Do we have a problem with the UI ACL?

Reply:
Not that I am aware of. We will just need to test this before we remove the "tech preview" label.

Reply:
sure, we should let QA went over all of these items and let them explicitly test it.